Form-Controls/index.html

```diff
@@ -14,14 +14,67 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    
+
+  <section>
+        
+<label for="name">enter your name</label>
+<input type="text" id="name" name="name"><br>
+ </section>
+    <br>
+    <section>
+  <label for="email">enter your email</label>
+<input type="email" id="email" name="email">
+ </section>
+      <br>
+      <br>
+      <section>
+     <label for="colors">Choose your colors:</label>
+    <input type="colors" name="colors" id="colors">
```

Comment:
What is this input doing here? What is its function? Can you talk me through it?